reference/6/Microsoft.PowerShell.Management/Start-Process.md

```diff
@@ -128,6 +128,25 @@ Start-Process -FilePath "$env:comspec" -ArgumentList "/c dir `"%systemdrive%\pro
 Start-Process -FilePath "$env:comspec" -ArgumentList "/c","dir","`"%systemdrive%\program files`""
 ```
 
+### Example 8: Create a detached process on Linux
+
+On Windows, `Start-Process` creates an independent process in a new console window that remains
+running independently of the launching shell. On non-Windows platforms, the newly started process is
+attached to the shell that launched. If the launching shell is closed, the child process is
+terminated.
+
+To avoid terminating the child process on Unix-like platforms, you can combine `Start-Process` with
+`nohup`. The following example launches a background PowerShell instance that stays alive even after
```

Comment:
I might add that nohup is short for "no hangup". And I think the description can be improved.  The example doesn't launch a background PowerShell instance, it launches nohup as a background instance that through parameters launches a PowerShell instance (which isn't a child of the original shell process).

Reply:
fixed